EU4ToVic2Tests/MapperTests/Buildings/BuildingTests.cpp

```diff
@@ -0,0 +1,62 @@
+#include "Buildings/Building.h"
+#include "gtest/gtest.h"
+
+TEST(Mappers_BuildingTests, costDefaultsToZero)
+{
+	std::stringstream input;
+	const mappers::Building theBuilding(input);
+
+	ASSERT_EQ(0, theBuilding.getCost());
+}
+
+TEST(Mappers_BuildingTests, costCanBeSet)
+{
+	std::stringstream input;
+	input << "cost = 100\n";
+	const mappers::Building theBuilding(input);
+
+	ASSERT_EQ(100, theBuilding.getCost());
+}
+
+TEST(Mappers_BuildingTests, modifiersDefaultToEmpty)
+{
+	std::stringstream input;
+	const mappers::Building theBuilding(input);
+
+	ASSERT_TRUE(theBuilding.getModifier().getAllEffects().empty());
+}
+
+TEST(Mappers_BuildingTests, modifiersCanBeAdded)
+{
+	std::stringstream input;
+	input << "modifier = {\n";
+	input << "\taModifier = 0.5\n";
+	input << "\tanotherModifier = 0.75\n";
+	input << "}\n";
+	const mappers::Building theBuilding(input);
+
+	ASSERT_EQ(2, theBuilding.getModifier().getAllEffects().size());
+	ASSERT_EQ(0.5, theBuilding.getModifier().getAllEffects().at("aModifier"));
```

Comment:
ASSERT_NEAR

Reply:
@Idhrendur Thoughts? If we set a number and ten compare it (doing no math at all), shouldn't EQ work as expected and always?

Reply:
The ways of IEEE 754 are not to be trusted. In this case it'll work fine because 0.1b and .11b have no loss of precision, but deities help us if you had gone for 0.2 or something.

Ahem. Best to use ASSERT_NEAR, if only to train your muscle memory for test writing.